webpack.config.js

```diff
@@ -32,9 +32,6 @@ const config = {
         ],
         use: {
           loader: 'babel-loader',
-          query: {
```

Comment:
`query` does not exist in webpack 5. Furthermore, the lodash plugin is already configured in `.babelrc` so we do not need to re-iterate it in the webpack config.